tests/test_spatialstats.py

```diff
@@ -397,6 +458,97 @@ def test_neff_exact_and_approx_hugonnet(self):
         # Check that the approximation is about the same as the original estimate within 10%
         assert neff_approx == pytest.approx(neff_exact, rel=0.1)
 
+    def test_number_effective_samples(self):
+        """Test that the wrapper function for neff functions behaves correctly and that output values are robust"""
+
+        raster = load_ref_and_diff()[0]
```

Comment:
I would combine the two lines with:
`raster, _, outlines = load_ref_and_diff()`
to avoid running the whole function twice.

Reply:
Done